cfme/tests/v2v/test_v2v_migrations_ui.py

```diff
@@ -186,9 +164,9 @@ def test_v2v_ui_set1(appliance, providers, enable_disable_migration_ui,
         ['mappings'][0]['target'])
 
     # Test Datacenter name in source and destination mapping select list:
-    soft_assert(providers[0].data['datacenters'][0]
+    soft_assert(v2v_providers[0].data['datacenters'][0]
```

Comment:
Formatting of the 'in' line appears off here. Suggestion:
```python
    soft_assert(
        v2v_providers[1].data['datacenters'][0] in view.form.cluster.target_clusters.all_items[0]
    )
```

Reply:
yeah not sure why that was like that, saw your other comment and then pushed a change and saw this suggested format. didn't fit on 100, split on in